app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.__class__.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_data: list) -> list:
+    people = []
+    for person_dict in people_data:
+        name = person_dict["name"]
+        age = person_dict["age"]
```

Comment:
better to use `get`
```suggestion
        name = person_dict.get("name")
        age = person_dict("age")
```
fix everywhere